src/logging.rs

```diff
@@ -0,0 +1,17 @@
+extern crate fern;
+
+pub fn bootstrap_logging() {
+    fern::Dispatch::new()
+        .format(|out, message, record| {
+            out.finish(format_args!(
+                "{}[{}] {}",
+                chrono::Utc::now().format("[%+]"),
+                record.level(),
+                message
+            ))
+        })
+        .level(log::LevelFilter::Debug)
```

Comment:
Probably better to have default level as info. My console output just went nuts with debug level :smile: